src/main.rs

```diff
@@ -66,10 +70,31 @@ fn write_err(err: StratisError) -> StratisResult<()> {
 }
 
 fn main() {
-
-    let engine = Rc::new(RefCell::new(SimEngine::new()));
-    // TODO: add cmdline option to specify engine
-    //  let context = Rc::new(RefCell::new(Context::new()));
+    let matches = App::new("stratis")
+        .version(&crate_version!())
```

Comment:
I can't see why we'd need multiple versions. Opened https://github.com/stratis-storage/stratisd/issues/138